tracing/tracers/lightstep/lightstep.go

```diff
@@ -32,6 +33,13 @@ func InitTracer(opts []string) (opentracing.Tracer, error) {
 			}
 		case "token":
 			token = parts[1]
+		case "tag":
+			if len(parts) > 1 {
+				tags := strings.SplitN(parts[1], "=", 2)
```

Comment:
currently we ignore those tags that don't have the form of `foo=bar`. I'd prefer:

a) fail early
b) consider such tags as boolean tags with the value of `true`

What do you think?

Reply:
a) is ok if we do not want to simply ignore the error as we do right now
b) can create more weirdness to the enduser, which I don't like. For example if you have complex tag settings and forgot one =foo then where ever in the chain the tags are specified this can create unexpected results

Reply:
between a) and c), I prefer a) because c) requires manual troubleshooting

Reply:
pushed a changed for a)

Reply:
thx @vetinari